distmv/normal.go

```diff
@@ -354,3 +355,31 @@ func (n *Normal) setSigma() {
 		n.sigma.FromCholesky(&n.chol)
 	})
 }
+
+// TransformNormal transforms the vector src, generated from a multidimensional
+// standard normal, to one generated under the distribution specified by the
+// receiver. If dst is non-nil, the result will be stored into dst, otherwise
+// a new slice will be allocated. TransformNormal will panic if the length of
+// src is not the dimension of the Normal, or if dst is non-nil and len(src) !=
+// len(dst).
+func (n *Normal) TransformNormal(dst, src []float64) []float64 {
```

Comment:
If it's on the `*Normal` it doesn't need to be called `TransformNormal`, it can just be `Transform`.

The normal logic for determining whether something is a package function or method is mainly one of two things: 1) namespacing and 2) making generic code. The first is obvious, the second is when you want to use specific implementations to fit within a generic framework provided by package-level functions or type methods.

Here, it looks like the `Transform` is the generic interface and `*Normal` is the specific implementation. So, with the name alteration, I'd make this a method.